org.gaphor.Gaphor.appdata.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.gaphor.Gaphor.desktop</id>
+  <name>Gaphor</name>
+  <summary>Simple modeling tool</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <metadata_license>CC-BY-3.0</metadata_license>
+  <project_license>LGPL-2.0+</project_license>
+  <url type="homepage">https://github.com/gaphor/gaphor</url>
```

Comment:
Do you have any other url's like help, donation or bugtracker? it would be nice to have them here :)

Reply:
I added the url for help / docs :+1: